app/src/components/register/ArtistSelection.tsx

```diff
@@ -33,6 +33,7 @@ const ArtistSelection: React.FC = () => {
 
   const onArtistChange = (event: React.FormEvent<HTMLSelectElement & any>): void => {
     setField('artistId', event.currentTarget.value.toString());
+    setField('artistWallet', artists[event.currentTarget.value].wallet);
```

Comment:
I don't know if this is off by one as the artist id starts from 1 but the array indexes from 0

Reply:
it does appear to be, fixing now